extras/configure_gluster_collectd.yml

```diff
@@ -32,3 +32,43 @@
       service:
         name: collectd
         state: restarted
+
+- name: Install choose-master package to elect master node 
+  hosts: ovirt-engine
+  vars:
+    choose_master_rpm: "https://copr-be.cloud.fedoraproject.org/results/redara\
+                       /gluster-collectd/fedora-27-x86_64/00798077-choose-master\
+                       /choose-master-1.0.0-1.fc27.noarch.rpm"
```

Comment:
Have you tried the yum_repositories to add the copr as repo and just install the choose-master rpm without mentioning the version?

https://docs.ansible.com/ansible/2.5/modules/yum_repository_module.html

Reply:
No. I didnt tried.